From 6cd1b3fe71eb40623040d5d85de3f28ba18077c6 Mon Sep 17 00:00:00 2001
From: trungvanle <trung.le.xk@renesas.com>
Date: Wed, 24 Jan 2024 09:28:39 +0700
Subject: [PATCH] ext: bayersink: Remove EGL_PIXMAP_BIT while using mali

This commit is added to remove EGL_PIXMAP_BIT to support bayersink
for G2L (mali) due to unsupported egl pixmap.

Signed-off-by: trungvanle <trung.le.xk@renesas.com>
---
 ext/bayersink/gstbayersink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/bayersink/gstbayersink.c b/ext/bayersink/gstbayersink.c
index c70ff66f6..8846f3d35 100644
--- a/ext/bayersink/gstbayersink.c
+++ b/ext/bayersink/gstbayersink.c
@@ -253,7 +253,7 @@ create_egl_context(struct context_bayer2rgb *ctx)
   };
 
   EGLint const fb_attribs[] = {
-    EGL_SURFACE_TYPE, EGL_PBUFFER_BIT | EGL_PIXMAP_BIT,
+    EGL_SURFACE_TYPE, EGL_PBUFFER_BIT,
     EGL_CONFORMANT, EGL_OPENGL_ES2_BIT,
     EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
     EGL_COLOR_BUFFER_TYPE, EGL_RGB_BUFFER,
-- 
2.17.1

